frontend/src/components/collective/UnpaidExpenseItem.js

```diff
@@ -0,0 +1,46 @@
+import React, {PropTypes} from 'react';
+
+import Currency from '../Currency';
+
+const UnpaidExpenseItem = ({ 
```

Comment:
It would. Those names are all taken up from current flow. Once we clean that up, we can remove a bunch of these names.